src/pages/PaymentCyclePage.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import {
+  connect,
+  // useDispatch,
+  useSelector,
+} from 'react-redux';
+import {
+  Form, Helmet, useTranslations, useModulesManager, useHistory,
+} from '@openimis/fe-core';
+import { EMPTY_STRING, MODULE_NAME } from '../constants';
+import PaymentCycleHeadPanel from '../components/PaymentCycleHeadPanel';
+// import {
+//   fetchPaymentCycle, clearPaymentCycle,
+// } from '../actions';
+import PaymentCycleTab from '../components/PaymentCycleTab';
+
+const useStyles = makeStyles((theme) => ({
+  page: theme.page,
+}));
+
+function PaymentCyclePage({ paymentCycle, paymentCycleUuid }) {
+  const rights = useSelector((store) => store.core.user.i_user.rights ?? []);
+  const classes = useStyles();
+  //   const dispatch = useDispatch();
+  const modulesManager = useModulesManager();
+  const history = useHistory();
+  const { formatMessage, formatMessageWithValues } = useTranslations(MODULE_NAME, modulesManager);
+  const back = () => history.goBack();
+
+  const titleParams = (paymentCycle) => ({
+    code: paymentCycle?.code ?? EMPTY_STRING,
+  });
+
+  useEffect(() => {
+    if (paymentCycleUuid) {
```

Comment:
do we need in that case such conditional statement if nothing is done within this scope of code?

Reply:
done